#! /bin/sh /usr/share/dpatch/dpatch-run
## 99_gcc4.6-fix.dpatch by Tobias Grimm <tg@e-tobi.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fixes gcc 4.6 issue - const needs custom default ctor or initializer
## DP: See C++ standard 8.5/9

@DPATCH@
diff --git a/PLUGINS/src/skincurses/skincurses.c b/PLUGINS/src/skincurses/skincurses.c
index 84076cd..debc8f7 100644
--- a/PLUGINS/src/skincurses/skincurses.c
+++ b/PLUGINS/src/skincurses/skincurses.c
@@ -26,7 +26,7 @@ public:
   virtual void DrawText(cPixmap *Pixmap, int x, int y, const char *s, tColor ColorFg, tColor ColorBg, int Width) const {}
   };
 
-static const cCursesFont Font;
+static const cCursesFont Font = cCursesFont();
 
 // --- cCursesOsd ------------------------------------------------------------
 
